ts/client/src/accounts/oracle.ts

```diff
@@ -63,26 +64,47 @@ export class StubOracle {
 export function parseSwitchboardOracleV1(accountInfo: AccountInfo<Buffer>): {
   price: number;
   lastUpdatedSlot: number;
+  uiDeviation: number;
 } {
   const price = accountInfo.data.readDoubleLE(1 + 32 + 4 + 4);
   const lastUpdatedSlot = parseInt(
     accountInfo.data.readBigUInt64LE(1 + 32 + 4 + 4 + 8).toString(),
   );
-  return { price, lastUpdatedSlot };
+  const minResponse = accountInfo.data.readDoubleLE(1 + 32 + 4 + 4 + 8 + 8 + 8);
+  const maxResponse = accountInfo.data.readDoubleLE(
+    1 + 32 + 4 + 4 + 8 + 8 + 8 + 8,
+  );
+  return { price, lastUpdatedSlot, uiDeviation: maxResponse - minResponse };
+}
+
+export function switchboardDecimalToBig(sbDecimal: {
+  mantissa: BN;
+  scale: number;
+}): Big {
+  const mantissa = new Big(sbDecimal.mantissa.toString());
+  const scale = sbDecimal.scale;
+  const oldDp = Big.DP;
+  Big.DP = 20;
+  const result: Big = mantissa.div(new Big(10).pow(scale));
+  Big.DP = oldDp;
+  return result;
```

Comment:
where's this code from? I've compared it to rusts's Decimal::to_f64 - is that the right place?

Reply:
https://github.com/switchboard-xyz/solana-sdk/blob/7eeab51398a67abe08999ddc5c6b5e693537c1cb/javascript/sbv2-lite/src/index.ts#L173

Its picked up from here